richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/model/RichTextState.kt

```diff
@@ -186,13 +186,15 @@ class RichTextState internal constructor(
         codeColor: Color = Color.Unspecified,
         codeBackgroundColor: Color = Color.Unspecified,
         codeStrokeColor: Color = Color.Unspecified,
+        listIndent: Int = 38
```

Comment:
Here we need to have a special value to ignore which can be -1 in the case of Int.
So the default value here should be changed to -1.